corehq/apps/cloudcare/templates/cloudcare/config.html

```diff
@@ -20,6 +20,38 @@ <h2>{% trans 'Manage Web Apps Permissions' %}</h2>
       Please avoid using <em>both</em> this configuration and Roles & Permissions to manage access.
       Please contact <a href="mailto:{{ SUPPORT_EMAIL }}">CommCare Support</a> if you need assistance
       setting up access via Roles & Permissions.
+      <br/><br/>
+      <h5>Transitioning from the Deprecated Feature to Roles & Permissions</h5>
+
+      <p>For the simplest case where each mobile worker belongs to only one group, follow these steps:</p>
+
+      <ol>
+        <li>
+          <strong>Set up a role for each group</strong><br/>
+          Create a corresponding role in the Roles & Permissions system for every existing group.
+        </li>
+        <li>
+          <strong>Configure permissions for each role</strong><br/>
+          Ensure that each role is granted the same permissions and access to apps that were previously available to the corresponding group.
+        </li>
+        <li>
+          <strong>Assign mobile workers to their new roles</strong><br/>
+          Map each mobile worker to the appropriate role that matches their previous group membership.
```

Comment:
This could also be a link, to the mobile workers page.

Reply:
[1d67ff6](https://github.com/dimagi/commcare-hq/pull/35245/commits/1d67ff63b0e00ca6f9f5030666c960bd189624bc)